lib/test-collection.js

```diff
@@ -159,12 +169,17 @@ TestCollection.prototype._buildTestWithHooks = function (test) {
 	var beforeHooks = this._buildHooks(this.hooks.beforeEach, test.title, context);
 	var afterHooks = this._buildHooks(this.hooks.afterEach, test.title, context);
 
-	return [].concat(beforeHooks, this._buildTest(test, context), afterHooks);
+	var sequence = new Sequence([].concat(beforeHooks, this._buildTest(test, context), afterHooks), true);
+	if (this.hooks.afterEachAlways.length !== 0) {
+		var afterAlwaysHooks = new Sequence(this._buildHooks(this.hooks.afterEachAlways, test.title, context));
+		sequence = new Sequence([sequence, afterAlwaysHooks], false);
+	}
+	return sequence;
 };
 
 TestCollection.prototype._buildTests = function (tests) {
 	return tests.map(function (test) {
-		return new Sequence(this._buildTestWithHooks(test), true);
+		return this._buildTestWithHooks(test);
```

Comment:
seems that with only the change that move the new Sequence to _buildTestWithHooks, tests will failed